scorecards_test.go

```diff
@@ -0,0 +1,63 @@
+package opslevel_test
+
+import (
+	"testing"
+
+	ol "github.com/opslevel/opslevel-go/v2023"
+	"github.com/rocktavious/autopilot/v2022"
+)
+
+func TestCreateScorecard(t *testing.T) {
+	request := `{{ template "scorecard_create_request" }}`
+	response := `{{ template "scorecard_create_response" }}`
+
+	client := ABetterTestClient(t, "scorecards/create_scorecard", request, response)
+	sc, err := client.CreateScorecard(ol.ScorecardInput{
+		Name:  "new scorecard",
+		Owner: ol.IdentifierInput{Alias: "platform"},
+	})
+
+	autopilot.Ok(t, err)
+	autopilot.Equals(t, "new scorecard", sc.Name)
+	autopilot.Equals(t, "platform", sc.Owner.OnTeam.Alias)
+}
+
+func TestUpdateScorecard(t *testing.T) {
+	request := `{{ template "scorecard_update_request" }}`
+	response := `{{ template "scorecard_update_response" }}`
+
+	client := ABetterTestClient(t, "scorecards/update_scorecard", request, response)
+	sc, err := client.UpdateScorecard(ol.IdentifierInput{Id: "scorecard-id"}, ol.ScorecardInput{
+		Name:  "updated scorecard",
+		Owner: ol.IdentifierInput{Id: "team-id"},
+	})
+
+	autopilot.Ok(t, err)
+	autopilot.Equals(t, "updated scorecard", sc.Name)
+	autopilot.Equals(t, ol.ID("team-id"), sc.Owner.OnTeam.Id)
+}
+
```

Comment:
Looks like we are missing a test for `ListScorecards`

Reply:
Added!